integration/tctl_terraform_env_test.go

```diff
@@ -126,6 +128,7 @@ func TestTCTLTerraformCommand_ProxyJoin(t *testing.T) {
 func TestTCTLTerraformCommand_AuthJoin(t *testing.T) {
 	t.Parallel()
 	testDir := t.TempDir()
+	prometheus.DefaultRegisterer = metricRegistryBlackHole{}
 
```

Comment:
Is this safe to do in a parallel test?

Reply:
If we had tests relying on metrics (which we should, but we don't) this might break them as metrics will stop being collected. I'll send a fix on master.